server/linter/metadata/linter_metadata.go

```diff
@@ -0,0 +1,140 @@
+package metadata
```

Comment:
This file contains everything that is required to configure the linter, in the future this data could come from anywhere else like a remote content manager